sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/AssetFileTypeNotSupportedException.cs

```diff
@@ -39,18 +40,25 @@ public AssetFileTypeNotSupportedException(string message, Exception inner)
         {
         }
 
+        /// <inheritdoc />
+        public override void GetObjectData(SerializationInfo info, StreamingContext context)
+        {
+            Argument.AssertNotNull(info, nameof(info));
+
+            info.AddValue(nameof(AttemptedFileType), AttemptedFileType);
+            info.AddValue(nameof(SupportedAssetFileTypes), SupportedAssetFileTypes);
+
+            base.GetObjectData(info, context);
+        }
+
         internal AssetFileTypeNotSupportedException(AssetFileType assetFileType, IReadOnlyList<AssetFileType> supportedAssetFileTypes)
             : base($"The provided asset file type of \"{assetFileType}\" is unsupported by Azure Object Anchors for conversion. Supported file types: {string.Join(", ", supportedAssetFileTypes)}")
         {
             AttemptedFileType = assetFileType;
             SupportedAssetFileTypes = supportedAssetFileTypes;
         }
 
-        /// <summary>
-        /// An Exception thrown during an attempt to provide an unsupported asset file type in an asset conversion operation
-        /// </summary>
-        /// <param name="info">The SerializationInfo</param>
-        /// <param name="context">The StreamingContext</param>
+        /// <inheritdoc />
         protected AssetFileTypeNotSupportedException(SerializationInfo info, StreamingContext context)
```

Comment:
I think you would need to populate the properties from the SerializationInfo, e.g. https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/RequestFailedException.cs#L73